workshop/01_Navigation/01_Navigate_to_something_you_know/1.3_Navigate_to_class.md

```diff
@@ -1,24 +1,31 @@
-Navigate to Class
+# Navigate to Class
+
+> ### Navigate to Class
+> - Ctrl+N (Windows/Linux)
+> - Command+O (Mac OS X)
 
 Navigate to a class by (partial) class name.
 
-Ctrl+N (Windows/Linux)
-Command+O (Mac OS X)
+1. **Navigate to class**
+
+    Navigate to the Customer class by using *Navigate to Class* and then typing `Customer`. Open the file by hitting enter.
+
+2. **Use CamelHumps to navigate**
+
+    Navigate to the _CustomerRepository_ class using CamelHumps (type `CR` or `CusR`).
 
-1. Navigate to class.
-   Navigate to the Customer class. Open the file by hitting enter.
+3. **Use middle matching to navigate**
+    
+    Navigate to the _DocumentRepository_ class by searching for `tRepo`.
 
-2. Use CamelHumps to navigate.
-   Navigate to the CustomerRepository class using CamelHumps (type "CR" or "CusR").
+4. **Use wildcards**
 
-3. Use middle matching to navigate.
-   Navigate to the DocumentRepository class by searching for "Repo".
+    Navigate to the _SettingsRepository_ by searching for `S*Repo`.
 
-4. Use wildcards.
-   Navigate to the SettingsRepository by searching for "S*Repo".
+5. **Use filters** 
 
-5. Filter to only check PHP files by clicking the filter icon.
+    Filter to only check PHP files by clicking the ![filter](https://www.jetbrains.com/help/img/idea/2017.1/filter.png) icon.
```

Comment:
I am investigating getting all the icons from the docs team and bundling them into the workshop so we can reference locally, I think this is OK for now.